src/services/config_extraction.rs

```diff
@@ -217,7 +225,7 @@ impl ConfigExtractionService for ConfigExtractionServiceImpl {
         let suite = self.evg_config_utils.find_suite_name(task_def).to_string();
         let task_tags = self.evg_config_utils.get_task_tags(task_def);
         let require_multiversion_setup = task_tags.contains(MULTIVERSION);
-        let no_multiversion_iteration = task_tags.contains(NO_MULTIVERSION_ITERATION);
+        let no_multiversion_generate_tasks = task_tags.contains(NO_MULTIVERSION_GENERATE_TASKS);
```

Comment:
@stevegrossmongodb, this is how I propagated a "tag" from the source evergreen task config to a `ResmokeGenParams` instance.